app.rb

```diff
@@ -0,0 +1,86 @@
+require './book'
+require './teacher'
+require './student'
+require './rental'
+
+class App
+  attr_accessor :books, :teachers, :students, :rentals
+
+  def initialize
+    @books = []
+    @teachers = []
+    @students = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each_with_index { |book, n| print "(#{n}) Title: \"#{book.title}\", Author: \"#{book.author}\"\n" }
+  end
+
+  def list_people
+    people = [*@teachers, *@students]
+    people.each_with_index { |person, i| print "(#{i}) Name: \"#{person.name}\", Age: \"#{person.age}\"\n" }
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [input a number]:'
+    input = gets.chomp.to_i
+
+    case input
+    when 1
+      print 'Age:'
+      age = gets.chomp.to_i
+      print 'Name:'
+      name = gets.chomp
+      print 'Enter Classroom:'
+      classroom = gets.chomp
+      new_student = Student.new(age, classroom)
+      new_student.name = name
+      @students << new_student
+      puts "#{new_student.name} added successfully"
+
+    when 2
+      print 'Age:'
+      age = gets.chomp.to_i
+      print 'Name:'
+      name = gets.chomp
+      print 'Specialization:'
+      specialization = gets.chomp
+      new_teacher = Teacher.new(age, name, specialization)
+      @teachers << new_teacher
+      puts "#{new_teacher.name} added successfully"
+    end
+  end
+
+  def create_book
+    print 'Book Title:'
+    title = gets.chomp
+    print 'Book Author:'
+    author = gets.chomp
+    new_book = Book.new(title, author)
+    @books << new_book
+    puts "#{new_book.title} created successfully"
+  end
+
+  def create_rental
```

Comment:
- [ ] When creating the rental an error produced, kindly fix it so the user can creat a rental with user id

![Screenshot_20220909_191617](https://user-images.githubusercontent.com/25227022/189417907-3795348c-274b-4b5b-811b-3d4a9bf7406e.png)

Reply:
rectified. Thanks